sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientIntegrationTest.java

```diff
@@ -78,73 +69,7 @@
 @Execution(ExecutionMode.SAME_THREAD)
 public class ServiceBusAdministrationClientIntegrationTest extends TestProxyTestBase {
     private static final Duration TIMEOUT = Duration.ofSeconds(20);
-
-    /**
-     * Test to connect to the service bus with an azure identity TokenCredential.
-     * com.azure.identity.ClientSecretCredential is used in this test.
-     * ServiceBusSharedKeyCredential doesn't need a specific test method because other tests below
-     * use connection string, which is converted to a ServiceBusSharedKeyCredential internally.
-     */
-    @Test
-    void azureIdentityCredentials() {
```

Comment:
Same comment as above

Reply:
Good point. Now that our tests run on federated by default, I renamed this special test to "azureClientSecretCredential"

Reply:
this one test needs an auxiliary tenant to work with secret. Federated Managed Identity and secret cannot co-exist. Have to enable this one test once aux-tenant is there